src/main/java/org/fenixedu/academic/dto/student/StudentStatuteBean.java

```diff
@@ -84,10 +83,8 @@ public String getEndPeriodFormatted() {
     }
 
     public String getDescription() {
-        return BundleUtil.getString(Bundle.ENUMERATION, getStatuteType().getDeclaringClass().getSimpleName() + "."
-                + getStatuteType().name())
+        return getStatuteType().getName().getContent(I18N.getLocale())
```

Comment:
Beware that invoking [`getContent(I18N.getLocale())`](https://github.com/FenixEdu/fenixedu-commons/blob/v1.1.0/src/main/java/org/fenixedu/commons/i18n/LocalizedString.java#L461-L475) is different than just invoking [`getContent()`](https://github.com/FenixEdu/fenixedu-commons/blob/v1.1.0/src/main/java/org/fenixedu/commons/i18n/LocalizedString.java#L480-L496).